frontend/src/component/layout/MainLayout/NavigationSidebar/NavigationSidebar.tsx

```diff
@@ -83,25 +87,65 @@ const EnterprisePlanBadge = () => (
     </Tooltip>
 );
 
-const FullListItem: FC<{ href: string; text: string; badge?: ReactNode }> = ({
+const FullListItem: FC<{
+    href: string;
+    text: string;
+    badge?: ReactNode;
+    onClick?: () => void;
+}> = ({ href, text, badge, onClick, children }) => {
+    return (
+        <ListItem disablePadding onClick={onClick}>
+            <ListItemButton dense={true} component={Link} to={href}>
+                <ListItemIcon sx={(theme) => ({ minWidth: theme.spacing(4) })}>
+                    {children}
+                </ListItemIcon>
+                <ListItemText primary={text} />
+                {badge}
+            </ListItemButton>
+        </ListItem>
+    );
+};
+
+const ExternalFullListItem: FC<{ href: string; text: string }> = ({
```

Comment:
Deliberate copy paste of the FullListItem but with open in new tab semantics (to avoid building abstractions).